lib/index.js

```diff
@@ -175,7 +185,7 @@ class RustLanguageClient extends AutoLanguageClient {
           )
         }
 
-        return cp.spawn("rustup", ["run", "nightly", "rls"], {
+        return cp.spawn("rustup", ["run", "rls"], {
```

Comment:
I don't think the `<toolchain>` argument is optional. `rustup run rls` does not work for me

Reply:
so `rustup run` does not use the default toolchain?